GDevelop.js/TestUtils/CodeGenerationHelpers.js

```diff
@@ -52,6 +52,197 @@ function generateCompiledEventsForEventsFunction(
   return runCompiledEventsFunction;
 }
 
+const generatedEventsCodeToJSFunction = (code, gdjs, runtimeScene) => {
```

Comment:
Would it be worth using this in the existing `generateCompiledEventsForEventsFunction`?